docassemble/ALAffidavitOfIndigency/data/questions/affidavit_body.yml

```
@@ -616,7 +616,13 @@ attachment:
     skip undefined: True
     fields:
       - "court": ${ affidavit_of_indigency_filing_court }
-      - "docket_number": ${case_name + '; ' + str(docket_numbers[0])}
+      - "docket_number": |
+          ${case_name } 
+          % if defined("docket_numbers"):
+          '; ' ${ docket_numbers[0] }
```

```suggestion
          ;  ${ docket_numbers[0] }
```
Don't need the quotes, they show up literally in the filled PDF.